docs/Simulation/FPGA-Accelerated-Simulators.rst

```diff
@@ -0,0 +1,64 @@
+FPGA-Accelerated Simulators
+==============================
+
+FireSim
+-----------------------
+
+`FireSim <https://fires.im/>`__ is an open-source cycle-accurate FPGA-accelerated full-system hardware simulation platform that runs on cloud FPGAs (Amazon EC2 F1).
+FireSim allows RTL-level simulation at orders-of-magnitude faster speeds than software RTL simulators.
+FireSim also provides additional device models to allow full-system simulation, including memory models and network models.
+
+FireSim currently supports running only on Amazon EC2 F1 FPGA-enabled virtual instances on the public cloud.
+In order to simulate your Chipyard design using FireSim, if you have not
+already, follow the initial EC2 setup instructions as detailed in the `FireSim
+documentation  <http://docs.fires.im/en/latest/Initial-Setup/index.html>`__.
+Then clone your full Chipyard repository onto your Amazon EC2 FireSim manager
+instance, and setup your Chipyard repository as you would normally.
+
+When you are ready to use FireSim, initalize it as library in Chipyard by running:
+
+.. code-block:: shell
+
+    # At the root of your chipyard repo
+    ./scripts/firesim-setup.sh --fast
+
+
+``firesim-setup.sh`` initializes additional submodules and then invokes
+firesim's ``build-setup.sh`` script. ``firesim-setup.sh`` accepts all of the same arguments and
+passes them through to ``build-setup.sh``, adding ``--library`` to properly
+initialize FireSim as a library submodule in chipyard. You may run
+``./sims/firesim/build-setup.sh --help`` to see more options.
+
+In order to build bitstreams, run simulations, or to generate MIDAS-transformed RTL for your
```

Comment:
bitstreams -> FPGA bitstreams ?
Also, I'm not sure we've mentioned MIDAS in this page, so the term "MIDAS-transformed RTL" might seem weird to a new user

Reply:
I would not explain it... rather just make this a work link to a place that does explain it (maybe the GitHub repo)?